core/src/test/java/dev/streamx/cli/run/MeshDefinitionResolverTest.java

```diff
@@ -31,25 +38,64 @@ void setup() {
     uut.parseResult = getParseResult();
   }
 
+  @AfterEach
+  void tearDown() throws IOException {
+    Files.deleteIfExists(CURRENT_DIRECTORY_MESH_YAML_PATH);
+    Files.deleteIfExists(CURRENT_DIRECTORY_MESH_YML_PATH);
+  }
+
+  @Test
+  void shouldResolveCurrentDirectoryMeshYaml() throws IOException {
+    // given
+    var meshYamlPath = CURRENT_DIRECTORY_MESH_YAML_PATH;
+    Files.copy(
```

Comment:
Can you do it some other way ? Creating resources in current dir during tests seems odd. I know it's cleaned in aftereach but still migh cause trouble. Are you sure this test would have write access wherever it's run ?

Reply:
extracted to `CurrentDirectoryProvider`